charts/modules/apps/monitoring/values.yaml

```diff
@@ -180,6 +180,20 @@ external-secrets:
             admin-password:
               awsSecretName: '{{ .Values.global.team }}-{{ .Values.global.environment }}-{{ .Values.global.deploymentName }}-grafanaadmin-creds'
               awsSecretKey: 'admin-password'
+      grafana-secrets:
```

Comment:
Maybe we name this grafana-loki-auth?

Reply:
It's a general-use secret with environment variables to be used in Grafana config. I.e. it can be used for anything, not just Loki.